index.html

```diff
@@ -0,0 +1,127 @@
+              Americans. "I saw how food changed them", he said. "All of this
+              left scars on me."
+            </li>
+            <li>
+              <span>1937</span> - Finishes university and takes a job in the US
+              Forestry Service
+            </li>
+            <li>
+              <span>1938</span> - Marries wife of 69 years Margret Gibson. Gets
+              laid off due to budget cuts. Inspired by Elvin Charles Stakman, he
+              returns to school study under Stakman, who teaches him about
+              breeding pest-resistent plants.
+            </li>
+            <li>
+              <span>1941</span> - Tries to enroll in the military after the
+              Pearl Harbor attack, but is rejected. Instead, the military asked
+              his lab to work on waterproof glue, DDT to control malaria,
+              disinfectants, and other applied science.
+            </li>
+            <li>
+              <span>1942</span> - Receives a Ph.D. in Genetics and Plant
+              Pathology
+            </li>
+            <li>
+              <span>1944</span> - Rejects a 100% salary increase from Dupont,
+              leaves behind his pregnant wife, and flies to Mexico to head a new
+              plant pathology program. Over the next 16 years, his team breeds
+              6,000 different strains of disease resistent wheat - including
+              different varieties for each major climate on Earth.
+            </li>
+            <li>
+              <span>1945</span> - Discovers a way to grown wheat twice each
+              season, doubling wheat yields
+            </li>
+            <li>
+              <span>1953</span> - crosses a short, sturdy dwarf breed of wheat
+              with a high-yeidling American breed, creating a strain that
+              responds well to fertilizer. It goes on to provide 95% of Mexico's
+              wheat.
+            </li>
+            <li>
+              <span>1962</span> - Visits Delhi and brings his high-yielding
+              strains of wheat to the Indian subcontinent in time to help
+              mitigate mass starvation due to a rapidly expanding population
+            </li>
+            <li><span>1970</span> - receives the Nobel Peace Prize</li>
+            <li>
+              <span>1983</span> - helps seven African countries dramatically
+              increase their maize and sorghum yields
+            </li>
+            <li>
+              <span>1984</span> - becomes a distinguished professor at Texas A&M
+              University
+            </li>
+          </ul>
+        </div>
+      </div>
+      <div class="info-section">
+        <p>
+          If you want to know more about Dr. Borlaug please
+          <a
+            id="tribute-link"
+            class="link"
+            href="https://en.wikipedia.org/wiki/Norman_Borlaug"
+            target="_blank"
+            >Click here</a
+          >
+        </p>
+      </div>
```

Comment:
Please use semantic HTML elements like `header`, `section`, `footer`,  and etc instead of  `div` tags. We have to use semantic HTML in our project because it makes the code easier to understand.